doc/developer.texinfo

```diff
@@ -2446,27 +2436,150 @@ GNU Texinfo Manual (@uref{http://www.gnu.org/software/texinfo/manual/texinfo/tex
 
 @need 1000
 @item
-Build everything to be distributed and copy it to the temporary staging area
-@c not @uref{...} because we don't want robots to notice and crawl the link
-(http://plse.cs.washington.edu/daikon/staging-daikon/).
-Part of this step is to build a set of publication abstracts.  If there are
-any failures, then investigate and correct as necessary.
-The final output of this step
-will be a list of files that were added/removed since the
-last release.
-If any of these differences is unexpected, then investigate and correct the
-difference, and redo the step.
+Once you have successfully reached this point, you are ready to make a release
+candidate and begin testing it. Any source changes required to fix a problem
+encountered above must be committed to the repository now.
+Verify there are no uncommitted files.
+
+@example
+git status -uno
+@end example
+
+@noindent
+The result of the git query above should be:
+
+@example
+On branch master
+Your branch is up-to-date with 'origin/master'.
+nothing to commit (use -u to show untracked files)
+@end example
+
+UNDONE: need to check doc/CHANGES time stamp.
+
+[Time: moments]
+
 
 @need 1000
+@item
+Set the version number in file @file{$DAIKONDIR/doc/VERSION}
+to an even number.
+You may do this manually, or if the value just needs to be incremented
+by 1 (to go from odd to even) you may use:
+
+@c "cd $DAIKONDIR" because the previous commands changed the working directory
+@example
+make update-dist-version-file
+@end example
+
+Optionally, override the distribution date (default: today) by
+redefining the environment variable @command{TODAY}:
+
+@example
+export TODAY='November 1, 2013'
+@end example
+
+Now, update the appropriate files with the date and version number
+of the release.
+
+@example
+make update-doc-dist-date-and-version
+@end example
+
+[Time: moments]
+
+UNDONE: we should combine and automate the step above and the
+step below.
+
+@need 1000
+@item
+Commit the version number changes back to the repository:
+
 @example
 cd $DAIKONDIR
-\rm -f `find . -type f -name '*~'`
+DAIKONVERSION=`cat $DAIKONDIR/doc/VERSION | xargs echo -n`
+git commit -a -m "Change version to $DAIKONVERSION"
+git push
+cd fjalar
+git commit -a -m "Change version to $DAIKONVERSION"
+git push
+@end example
+
+[Time: moments]
+
+
+@need 1000
+@item
+It is important to build the release candidate from a clean enlistment.
+This is to ensure we are testing only the files intended for release and
+that those files are the same as a user would receive with a clone of
+the repository. It is also recommend doing this step on your system's
```

Comment:
I would drop the last two sentences.  A reader should just follow your instructions, and so it will naturally be in /tmp/$USER.